src/helpers/ramlParser.js

```diff
@@ -103,14 +103,14 @@ function _parseRAML(filePath, dest, baseUri, listTraits, cb) {
     creator.createFilesSync(dest, result, 'utf-8');
     return cb('Generation went fine');
   }, (err) => {
-    if(err && err.message && err.message.indexOf('ECONNREFUSED') !== -1) {
-      logger.warning('Could not download traits.');
-      return cb();
+    if(err && err.message && err.message.indexOf('ECONNREFUSED') !== -1 || err && err.message && err.message.indexOf('ETIMEDOUT') !== -1) {
+      logger.error(`Could not download traits for: ${dest}`);
```

Comment:
Be more precise here, why we couldn't get traits? What was the error? Which trait? If this fail we need to know EXACTLY what happened